source/install/install-ubuntu-1404-postgresql.rst

```diff
@@ -0,0 +1,79 @@
+.. _install-ubuntu-1404-postgresql:
+
+Installing PostgreSQL Database Server
+=====================================
+
+Install and set up the database for use by the Mattermost server. You can install either PostgreSQL or MySQL. To install MySQL, see :ref:`install-ubuntu-1404-mysql`
+
+Assume that the IP address of this server is 10.10.10.1
+
+**To install PostgreSQL on Ubuntu Server 14.04:**
+
+1. Log into the server that will host the database and issue the following command:
+
+  ``sudo apt-get install postgresql postgresql-contrib``
+  
+  When the installation is complete, the PostgreSQL server is running, and a Linux user account called *postgres* has been created.
+
+2. Log into the *postgres* account. 
```

Comment:
We don't have a guideline for this yet. It seemed at the time that italicizing the user account name was appropriate.  Do you think this is something for the style guide?